packages/react/src/Panel/Panel.tsx

```diff
@@ -2,11 +2,6 @@ import { Paper, PaperProps } from '@mantine/core';
 import cx from 'clsx';
 import classes from './Panel.module.css';
 
-export interface PanelStylesParams {
```

Comment:
This appears to be carryover from Mantine 6.  It is not used anywhere else in the codebase.